examples/sara_work_area/genzsprtwo_39_files_oraganizer.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+"""
+A program to organize files into directories
+based on their extensions 
+"""
+
+parent_directory = "./dir"
+image_directory = "./dir/Images"
+text_directory = "./dir/Text"
+miscellaneous_directory = "./dir/Miscellaneous"
+
+for directory in [image_directory, text_directory, miscellaneous_directory]:
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+
+try:
+    no_of_img_files, no_of_txt_files, no_of_misc_files = 0, 0, 0
+    files = os.listdir(parent_directory)
```

Comment:
We can make use of the built-in glob library to specifically get *.jpg, *.png etc list of file paths and then we can create dirs and move the relevant files to specific dirs.

Reply:
i prefer using os module in this case